src/openrct2/management/Award.h

```diff
@@ -13,13 +13,7 @@
 
 #include <vector>
 
-struct Award
-{
-    uint16_t Time;
-    uint16_t Type;
-};
-
-enum class ParkAward : uint8_t
+enum class ParkAward : uint16_t
```

Comment:
Having `ParkAward` and `Award` is quite confusing. Would be better to rename to `AwardType`.

May be done in a later PR, though.